package.json

```diff
@@ -41,9 +41,9 @@
 		"tiny-glob": "^0.2.9",
 		"typescript": "^4.9.4"
 	},
-	"packageManager": "pnpm@7.30.3",
+	"packageManager": "pnpm@8.0.0",
 	"engines": {
-		"pnpm": "^7.0.0"
+		"pnpm": "^8.0.0"
```

Comment:
this would break on stackblitz, that is currently on 7.13.2

Reply:
no longer!